### PR TITLE
[REG 2.066] Issue 15674 - alias this rejected for 'out' parameter

### PR DESCRIPTION
**Pull request by 9rnsr**

### CODE REVIEW

File: test/runnable/overload.d [hunk 1]
```diff
@@ -1025,6 +1025,28 @@ void test11915()
...
+    alias v this;
+}
+
+void test15674()
+{
+    Foo15674 f;
+    get15674(f);
+    assert(f == 42);
+    assert(f.v == 42);
+}
```
Comment (mathias-lang-sociomantic) [inline test/runnable/overload.d]: Would it make sense to mirror [the tests for issue 7945](https://github.com/D-Programming-Language/dmd/blob/969e0903a66d06331785495cd88032fa6d19eca6/test/runnable/aliasthis.d#L1190-L1197) ?

**Comment by mathias-lang-sociomantic** [comment]
Sweet, thanks a lot. And :+1: for targeting stable.
FYI @mihails-strasuns-sociomantic

File: test/runnable/overload.d (hunk 1 above)
Comment (9rnsr) [inline test/runnable/overload.d]: Nice catch. `runnable/aliasthis.d` is better place for the 15674 test.

**Comment by mathias-lang-sociomantic** [comment]
@MartinNowak : Any chance you could take a look at this ?

**Comment by MartinNowak** [comment]
Auto-merge toggled on

**Pull request merged**